etc/nginx/nginx.conf

```diff
@@ -0,0 +1,52 @@
+user  nginx;
+worker_processes 1;
+
+# This log file is mapped to stderr in the nginx base image.
+error_log  /var/log/nginx/error.log warn;
+
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] '
+        '"$request" $status $body_bytes_sent '
+        '"$http_referer" "$http_user_agent" "$http_x_forwarded_for" '
+        '$request_time $upstream_response_time';
+
+    # This log file is mapped to stdout in the nginx base image.
+    access_log  /var/log/nginx/access.log  main;
+
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+
+    upstream django {
+        server webssl:8000;
+    }
+
+    server {
+        listen 80;
+        charset utf-8;
+        return 301 https://$host$request_uri;
```

Comment:
This should not be `301` in dev. Otherwise, when the user run with non ssl, the browser will redirect to `https`